components/floating_buttons/FilterRuns.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <b-button v-b-tooltip.hover.left="tooltip" pill size="lg" :variant="variant" @click="filterRuns"
```

Comment:
Time selectors are a bit too small right now
```suggestion
  <b-button v-b-tooltip.hover.left="tooltip" pill size="xl" :variant="variant" @click="filterRuns"
```